app/views/devise/registrations/new.html.haml

```diff
@@ -44,8 +44,20 @@
     = f.telephone_field :phone_number, class: 'form-control', required: 'true'
 
   .col-sm-8.field
```

Comment:
Can we put a big label here for "Tell us where you want to drive (does not need to be near your home)" or something like that. Even now, for our devprogress folks looking we want this to be clear.